tests/foreman/cli/test_hostgroup.py

```diff
@@ -669,6 +681,91 @@ def test_negative_create_with_content_source(self):
                 'organization-ids': self.org['id'],
             })
 
+    @run_only_on('sat')
+    @tier2
+    @upgrade
+    def test_positive_create_with_synced_content(self):
+        """Check if hostgroup with synced kickstart repository can be created
+
+        :id: 7c51ac72-359c-488a-8658-88b5a94d7e7a
+
+        :expectedresults: Hostgroup should be created and has proper
+            installation content id present
+
+        :BZ: 1415707
+
+        :CaseLevel: Integration
+        """
+        # Common entities
+        org = make_org()
+        lce = make_lifecycle_environment({'organization-id': org['id']})
+        product = make_product({'organization-id': org['id']})
+        repo = make_repository({
+            u'url': settings.rhel6_os,
```

Comment:
My only concern is - what if `rhel6_os` is not set? Unfortunately we can't use `skip_if_not_set` as there's no separate section for `rhel6_os`, but maybe we could add extra check here and skip or fail the test in case it's not set as there's no point in proceeding?